mne/io/nicolet/nicolet.py

```diff
@@ -0,0 +1,207 @@
+        Names of channels or list of indices that should be designated
+        EOG channels. If None (default), the channel names beginning with
+        ``EOG`` are used.
+    ecg : list or tuple
+        Names of channels or list of indices that should be designated
+        ECG channels. If None (default), the channel names beginning with
+        ``ECG`` are used.
+    emg : list or tuple
+        Names of channels or list of indices that should be designated
+        EMG channels. If None (default), the channel names beginning with
+        ``EMG`` are used.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated
+        MISC channels. If None, (default) none of the channels are designated.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    raw : Instance of Raw
+        A Raw object containing the data.
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+    """
+    return RawNicolet(input_fname, montage=montage, eog=eog, emg=emg,
+                      misc=misc, preload=preload, verbose=verbose)
+
+
+def _get_nicolet_info(fname, eog, ecg, emg, misc):
+    """Function for extracting info from Nicolet header files."""
+    info = _empty_info()
+    fname = fname.split('.')[0]
+    header = fname + '.head'
+    if not path.exists(header):
+        raise IOError('Cannot find header file %s.' % header)
+
+    logger.info('Reading header...')
+    header_info = dict()
+    with open(header, 'r') as fid:
+        for line in fid:
+            var, value = line.split('=')
+            if var == 'elec_names':
+                value = value[1:-2].split(',')  # strip brackets
+            elif var == 'conversion_factor':
+                value = float(value)
+            elif var != 'start_ts':
+                value = int(value)
+            header_info[var] = value
+
+    ch_names = header_info['elec_names']
+    if eog is None:
+        eog = [idx for idx, ch in enumerate(ch_names) if ch.startswith('EOG')]
+    if ecg is None:
+        ecg = [idx for idx, ch in enumerate(ch_names) if ch.startswith('ECG')]
+    if emg is None:
+        emg = [idx for idx, ch in enumerate(ch_names) if ch.startswith('EMG')]
+    if misc is None:
+        misc = list()
+
+    date, time = header_info['start_ts'].split()
+    date = date.split('-')
+    time = time.split(':')
+    sec, msec = time[2].split('.')
+    date = datetime.datetime(int(date[0]), int(date[1]), int(date[2]),
+                             int(time[0]), int(time[1]), int(sec), int(msec))
+    info.update({'filename': fname, 'nchan': header_info['num_channels'],
+                 'meas_date': calendar.timegm(date.utctimetuple()),
+                 'sfreq': header_info['sample_freq'], 'ch_names': ch_names,
+                 'description': None, 'buffer_size_sec': 10.})
+
+    cal = header_info['conversion_factor'] * 1e-6
```

Comment:
Here I'm just guessing that the data is in micro volts. Seems like the right scale, if you look at the data.